modules/porous_flow/doc/tests/radialinjection.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import matplotlib.pyplot as plt
+import pandas as pd
```

Comment:
Yeah, just checked and `pandas` is not in the MOOSE miniconda package. My default python is a different installation, so I didn't notice this. I can change this to use `numpy.genfromtxt`. I just used `pandas` because it is what I always use to read `csv` files.